modules/statsd/manifests/init.pp

```diff
@@ -19,12 +19,7 @@
   validate_bool($manage_repo_class)
 
   include nodejs
-
-  if $manage_repo_class {
```

Comment:
If the use of this conditional is being removed, it would be good to also remove the parameter, and any use of that in the hieradata as well.

Reply:
Good shout, done 👍

Reply:
Wonderful :)